saleor/graphql/order/mutations/orders.py

```diff
@@ -164,13 +164,30 @@ def clean_input(cls, info, instance, data):
                 cleaned_input[key] = draft_order_cleaned_input[key]
         return cleaned_input
 
+    @classmethod
+    def get_instance(cls, info, **data):
+        # call get_instance from ModelMutation
+        instance = super(DraftOrderUpdate, cls).get_instance(info, **data)
```

Comment:
The issue is multi level inheritance here
(`OrderUpdate` <- `DraftOrderUpdate` <- `DraftOrderCreate` <- `ModelMutation`)
we have `get_instance` defined on `ModelMutation`, overwritten on `DraftOrderUpdate` and then here (which is where we wanna call one from ModelMutation, ommiting the `DraftOrderUpdate`)
Solution works but it's kinda hackish as it changes MRO
Maybe it's time to remove inheritance on mutations `orderUpdate` and `orderDraftUpdate`? Looking for opinions.

Reply:
I'm fine with removing inheritance if it's causing trouble here. Looks like `OrderUpdate` shouldn't inherit from draft order mutations.